manifests/charts/aeraki/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: aeraki
+description: A Helm chart for Kubernetes
```

Comment:
Could we modify the default description to the introduction of Aeraki?

Reply:
ok